scripts/core/ui/components/SearchBar/index.tsx

```diff
@@ -85,14 +89,45 @@ export default class SearchBar extends React.Component<IProps, IState> {
     }
 
     render() {
-        const {timeout} = this.props;
+        const {debounced} = this.props;
         const {searchBarExtended} = this.state;
         const _uniqueId = this.state.uniqueId;
         const minLength = this.props.minLength ? this.props.minLength : 2;
+        const removeButton: React.ReactNode = (
+            <button
+                type="button"
+                className="search-close visible"
+                onClick={this.resetSearch}
+            >
+                <i className="icon-remove-sign" />
+            </button>
+        );
+        const showButtons = searchBarExtended && this.state.searchInputValue.trim().length > 0;
+        const actionButtons: React.ReactNode = (
+            debounced?.timeout != null && showButtons
```

Comment:
you probably don't care about timeout at this point. Only if `debounced` is null or not?